### DISCUSSION TITLE
Sharing my CSS for a good L&F

### DISCUSSION THREAD

**Discussion by ranjithk-78** [usememos/memos#4227] [Show and tell]
Hi Memos Team,
Thanks for a good piece of software :) Reminds me a lot of Docuwiki without any clutter
I have created a CSS to customize the L&F to make it look a bit better (at least to me :). Not sure if this is the right forum but sharing here with the community. Add it in **settings > system > general** 
[style.txt](https://github.com/user-attachments/files/18226180/style.txt)
Before:
![398076193-085dc4a9-04fb-485e-9ad3-f9d1ed21bd8d](https://github.com/user-attachments/assets/c996ad60-b176-4805-9987-3ea746432109)
After:
![398078879-61a6e525-19ab-4d4c-8ccd-906504277a9b](https://github.com/user-attachments/assets/062cbe4b-8bc6-4a2d-9d2f-2eefdbfb6bf7)
Ranjith

**Comment by RoccoSmit** [comment]
Nice, there is a share and tell section on Memo's Discord ([here](https://discord.com/channels/1017113925776588902/1053904069745442846)) where you could share this with the community as well

**Comment by bizzarrone** [comment]
could yo uhelp me to grow the area for the notes?

**Reply by touro411** [reply]
you can add this, this works for me

 #root main > :not(aside) [class*="max-w-"],
  #root main > :not(aside) .mx-auto,
  #root main > :not(aside) .container {
    max-width: 100% !important;
    width: 100% !important;
  }

  /* Memo-Karten & Inhalte nur im Content-Bereich */
  #root main > :not(aside) .memo-content-wrapper,
  #root main > :not(aside) .memo-card,
  #root main > :not(aside) [data-role="memo"] {
    max-width: 100% !important;
    width: 100% !important;
  }

  /* Tailwind Typography-Prosa nur im Content-Bereich entsperren */
  #root main > :not(aside) .prose {
    max-width: 100% !important;
  }

**Comment by codecreate-dev** [comment]
Is it not just easier to add a new CSS file for custom changes, rather than adding it in the admin settings?
